runhouse/resources/hardware/launcher_utils.py

```diff
@@ -174,7 +174,7 @@ def _update_from_den_response(cls, cluster, config: dict):
             return
 
         for attribute in [
-            "launched_properties",
+            "compute_properties",
```

Comment:
maybe we check both for BC?

Reply:
Looks like this is outdated actually, closed the PR and will make a new one if I need to post-launcher updates